cache-extension-demo/SAM/LambdaCode/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "lambdacode",
```

Comment:
Can we rename to coming more meaningful?

Reply:
Modified it to "database-entry"